BACKLOG.txt

```diff
@@ -75,13 +75,13 @@ Need to create Spring web project for tomcat web server using
  +DEV-031: Create EditAdController and editAd.jsp to edit ads and RemoveAdController to
         remove an ad;+reznik
 
- DEV-032: Add autofill for city, phone, email fields with user's home city;
++DEV-032: Add autofill for city, phone, email fields with user's home city;+panarina
 
  DEV-033: Develop messaging system: message table, dao, service, jsp changes of
         editAd.jsp, handle messaging on editAd.jsp and controller DEP-031;
 
  DEV-034: Add column date to ad table and entity, sorting output ad collections by
-        that date;
+        that date;+panarina
```

Comment:
Add '+' here, DEV-034 into name of the pool request or remove change of this line;

Reply:
done (remove)